webpack.config.js

```diff
@@ -160,6 +193,27 @@ function getWebpackConfig( {
 					cacheIdentifier,
 					exclude: /node_modules\//,
 				} ),
+				{
+					test: /\.jsx?$/,
+					include: _.memoize( shouldTranspileDependency ),
+					use: [
+						{
+							loader: 'thread-loader',
+							options: {
+								workers: workerCount,
+							},
+						},
+						{
+							loader: 'babel-loader',
+							options: {
+								configFile: path.resolve( __dirname, 'babel.dependencies.config.js' ),
+								babelrc: false,
+								cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
+								cacheIdentifier,
+							},
+						},
+					],
+				},
```

Comment:
Could we use the `Transpile` shard from `@automattic/calypso-build` for this?

Reply:
possibly, i'll try it out

Reply:
done!